docs/src/main/sphinx/release/release-405.md

```diff
@@ -0,0 +1,219 @@
+
+## JDBC driver
+
+* Return correct values in `NULLABLE` columns of the
+  `DatabaseMetaData.getColumns` result. ({issue}`15214`)
+
+## BigQuery connector
+
+* Add support for using Arrow serialization when reading from BigQuery. This is
+  an experimental feature and can be enabled by setting the
+  `bigquery.experimental.arrow-serialization.enabled` catalog configuration
+  property to `true`. ({issue}`14972`)
+* Fix queries incorrectly executing with the project ID specified in the
+  credentials instead of the project ID specified in the `bigquery.project-id`
+  catalog property. ({issue}`14083`)
+
+## Delta Lake connector
+
+* Add support for views. ({issue}`11609`)
+* Improve performance and reduce storage requirements when running the `vacuum`
+  procedure on S3-compatible storage. ({issue}`15072`)
+* Improve performance of reading Parquet files for `boolean`, `tinyint`,
+  `short`, `int`, `long`, `float`, `double`, `short decimal`, `UUID`, `time`,
+  `decimal`, `varchar`, and `char` data types. The
+  `parquet.optimized-reader.enabled` catalog configuration property can be set
+  to `false` to disable the optimized implementation. ({issue}`14423`, {issue}`14667`)
+* Improve query performance when the `nulls fraction` statistic is not available
+  for some columns. ({issue}`15132`)
+* Improve performance when reading Parquet files. ({issue}`15257`)
+* Improve memory accounting for `INSERT`, `MERGE`, and `CREATE TABLE AS SELECT`
+  queries. ({issue}`14407`)
+* Improve performance of reading Parquet files for queries with filters. ({issue}`15268`)
+* Improve `DROP TABLE` performance for tables stored on AWS S3. ({issue}`13974`)
+* Improve performance of reading Parquet files for `timestamp` and
+  `timestamp with timezone` data types. ({issue}`15204`)
+* Allow registering existing table files in the metastore with the new
+  [`register_table` procedure](delta-lake-register-table). ({issue}`13568`)
+* Deprecate creating a new table with existing table content. This can be
+  re-enabled using the `delta.legacy-create-table-with-existing-location.enabled`
+  configuration property or the
+  `legacy_create_table_with_existing_location_enabled` session property. ({issue}`13568`)
+* Fix `DROP TABLE` leaving files behind when using managed tables stored on S3
+  and created by the Databricks runtime. ({issue}`13017`)
+* Fix query failure when the path contains special characters. ({issue}`15183`)
+
+## Google Sheets connector
+
+* Add support for setting a read timeout with the `gsheets.read-timeout`
+  configuration property. ({issue}`15322`)
+* Rename the `credentials-path` configuration property to
+  `gsheets.credentials-path`, `metadata-sheet-id` to
+  `gsheets.metadata-sheet-id`, `sheets-data-max-cache-size` to
+  `gsheets.max-data-cache-size`, and `sheets-data-expire-after-write` to
+  `gsheets.data-cache-ttl`. ({issue}`15042`)
+
+## Hive connector
+
+* Add support for referencing nested fields in union-typed columns. ({issue}`15278`)
```

Comment:
This is ambiguous. Trino doesn't have a union type. Maybe say "in columns with `UNIONTYPE` Hive type"